packages/notebook-extension/src/celltaglist.tsx

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (c) Jupyter Development Team.
+ * Distributed under the terms of the Modified BSD License.
+ */
+
+import { NotebookPanel } from '@jupyterlab/notebook';
+import React from 'react';
+//import PropTypes from 'prop-types';
+import { ReactWidget, VDomModel, VDomRenderer } from '@jupyterlab/apputils';
+import { CommandRegistry } from '@lumino/commands';
+import { Button } from '@jupyterlab/ui-components';
+
+export class FilterButtonWidget extends ReactWidget {
+  private _commands: CommandRegistry;
+  private _commandID: string;
+
+  constructor(commands: CommandRegistry, commandID: string) {
+    super();
+    this._commands = commands;
+    this._commandID = commandID;
+  }
+  onClick = () => this._commands.execute(this._commandID);
+  render() {
+    return (
+      <>
+        <p> Filter cells with tags</p>
+        <Button onClick={this.onClick} />
```

Comment:
```suggestion
        <p>{this._trans.__('Filter cells with tags')}</p>
        <Button onClick={this.onClick}>{this._commands.label(this._commandID)}</Button>
```

where `_trans` needs to be threaded in the constructor separately.

Reply:
This is what I think should work because the Button accepts the [`children` props](https://react.dev/learn/passing-props-to-a-component#passing-jsx-as-children)

Reply:
@krassowski Ok thanks. The `FilterButtonWidget` has been updated taken this into account.